Cargo.toml

```diff
@@ -17,11 +17,10 @@ futures = { version = "0.3", default-features = false, features = ["std"] }
 gltf = { version = "0.15", features = ["extras", "names"] }
 log = "0.4"
 image = "0.23"
+num_enum = "0.5.1"
```

Comment:
where is it used?

Reply:
it is used to simply convert winit KeyCode to dotrix KeyCode

Reply:
should be removed if we convert KeyCode manually